### DISCUSSION TITLE
Doubt on using datasets with diffrent box sizes from periodic DFT calculation.

### DISCUSSION THREAD

**Discussion by magnamad** [deepmodeling/deepmd-kit#3079] [Q&A]
Hi ,

I am trying to devolepe a NN potential for bulk gas phase reactions. I have used VASP AIMD simulations to generate the training data . Since I need to capture different subsets of reactions , I have different AIMD runs with different box sizes. Is there a theoretical restriction in including systems with different box sizes in the training dataset ? I think it shouldn't matter as long as PBC is enabled.

Regards ,
magnamad

**Comment by njzjz** [comment]
There's no theoretical restriction.
